sci-libs/gdal/gdal1-1.11.5.recipe

```diff
@@ -116,26 +124,73 @@ BUILD_PREREQUIRES="
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
-defineDebugInfoPackage gdal1$secondaryArchSuffix \
-	$libDir/libgdal.so.$libVersion
+debugList=("$libDir"/libgdal.so.$libVersion)
+if [ -z "$commandSuffix" ]; then
+debugList+=(
+	"$commandBinDir"/gdal_contour \
+	"$commandBinDir"/gdal_grid \
+	"$commandBinDir"/gdal_rasterize \
+	"$commandBinDir"/gdal_translate \
+	"$commandBinDir"/gdaladdo \
+	"$commandBinDir"/gdalbuildvrt \
+	"$commandBinDir"/gdaldem \
+	"$commandBinDir"/gdalenhance \
+	"$commandBinDir"/gdalinfo \
+	"$commandBinDir"/gdallocationinfo \
+	"$commandBinDir"/gdalmanage \
+	"$commandBinDir"/gdalserver \
+	"$commandBinDir"/gdalsrsinfo \
+	"$commandBinDir"/gdaltindex \
+	"$commandBinDir"/gdaltransform \
+	"$commandBinDir"/gdalwarp \
+	"$commandBinDir"/nearblack \
+	"$commandBinDir"/ogr2ogr \
+	"$commandBinDir"/ogrinfo \
+	"$commandBinDir"/ogrlineref \
+	"$commandBinDir"/ogrtindex \
+	"$commandBinDir"/testepsg
+	)
+fi
+defineDebugInfoPackage gdal1$secondaryArchSuffix "${debugList[@]}"
 
 BUILD()
 {
 	expatDir=$portPackageLinksDir/lib~libexpat
 	autoreconf -fi
-	runConfigure ./configure --with-expat=$expatDir
+	runConfigure --omit-dirs "binDir dataDir" ./configure \
+		--bindir "$commandBinDir" \
+		--datadir="$dataDir"/gdal1 \
+		--with-expat=$expatDir
 	make $jobArgs
 }
 
 INSTALL()
 {
 	make install
 
+	if [ -n "$secondaryArchSuffix" -a -z "$commandSuffix" ]; then
+		mkdir -p "$binDir"
+		mv "$commandBinDir"/gdal-config "$binDir"
+	fi
+
 	prepareInstalledDevelLib libgdal
 	fixPkgconfig
 
 	# devel package
 	packageEntries devel \
-		$binDir/gdal-config \
-		$developDir
+		"$binDir"/gdal-config \
+		"$developDir"
+
+	if [ -n "$secondaryArchSuffix" -a -z "$commandSuffix" ]; then
+		rmdir "$binDir"
+	fi
+
+	if [ -n "$commandSuffix" ]; then
+		rm -rf "$binDir"
+	fi
+}
+
+TEST()
+{
+	make check
```

Comment:
There is no such `check` target :(
It seems some tests are being done at build time, as we can see in the `GNUmakefile`:
```
default:        lib-target apps-target swig-target gdal.pc

lib-target:     check-lib;
```
(I did not investigate further.)
Feel free to drop the entire `TEST` bloc (or fix it ;)

Reply:
Forgot to pull this out, added it to trigger haikuporter from time to time to run the configure script, will be left out :)